Dockerfile

```diff
@@ -81,7 +81,7 @@ RUN if [ "$include_dagmc" = "true" ] ; \
     mkdir MOAB ; \
     cd MOAB ; \
     mkdir build ; \
-    git clone  --single-branch --branch develop https://bitbucket.org/fathomteam/moab/ ; \
+    git clone  --single-branch --branch develop https://bitbucket.org/fathomteam/moab.git ; \
```

Comment:
Let's use the `master` branch of MOAB or a specific version. Their `develop` branch is experimental.